ddtrace/contrib/botocore/patch.py

```diff
@@ -122,15 +122,43 @@ def inject_trace_data_to_message_attributes(trace_data, entry, endpoint=None):
         log.warning("skipping trace injection, max number (10) of MessageAttributes exceeded")
 
 
-def inject_trace_to_sqs_or_sns_batch_message(params, span, endpoint=None):
-    # type: (Any, Span, Optional[str]) -> None
+def get_queue_name(params):
+    # type: (str) -> str
+    """
+    :params: contains the params for the current botocore action
+
+    Return the name of the queue given the params
+    """
+    queue_url = params["QueueUrl"]
+    url = urlparse(queue_url)
+    return url.path.rsplit("/", 1)[-1]
```

Comment:
This could be overkill but we can use os.path.split(url.path) to get the queue name